src/test/javascript/spec/integration/code-editor/code-editor-container.integration.spec.ts

```diff
@@ -302,46 +288,46 @@ describe('CodeEditorContainerIntegration', () => {
         containerFixture.detectChanges();
 
         // container
-        expect(container.commitState).to.equal(CommitState.COULD_NOT_BE_RETRIEVED);
-        expect(container.editorState).to.equal(EditorState.CLEAN);
-        expect(container.buildOutput.isBuilding).to.be.false;
-        expect(container.unsavedFiles).to.be.empty;
+        expect(container.commitState).toEqual(CommitState.COULD_NOT_BE_RETRIEVED);
+        expect(container.editorState).toEqual(EditorState.CLEAN);
+        expect(container.buildOutput.isBuilding).toBe(false);
+        expect(container.unsavedFiles).toBeEmpty();
 
         // file browser
-        expect(checkIfRepositoryIsCleanStub).to.have.been.calledOnce;
-        expect(getRepositoryContentStub).to.not.have.been.calledOnce;
-        expect(container.fileBrowser.errorFiles).to.deep.equal(extractedErrorFiles);
-        expect(container.fileBrowser.unsavedFiles).to.be.empty;
+        expect(checkIfRepositoryIsCleanStub).toHaveBeenCalledTimes(1);
+        expect(getRepositoryContentStub).not.toHaveBeenCalled;
```

Comment:
```suggestion
        expect(getRepositoryContentStub).not.toHaveBeenCalled();
```
Looks like the function call is omitted.

Reply:
Thanks for pointing this out. I somehow missed this. I adapted it.